apisix/plugins/hmac-auth.lua

```diff
@@ -316,9 +317,17 @@ local function get_params(ctx)
     local signed_headers = core.request.header(ctx, signed_headers_key)
     core.log.info("signature_key: ", signature_key)
 
+    core.request.set_header(access_key, nil)
```

Comment:
Why delete these request headers?  Are there any protocol specifications?

Reply:
These request headers are related to the `hmac-auth` plugin, and upstream does not need to care about their existence.